### PR TITLE
Create resolv.conf in chroot if required

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -16,13 +16,38 @@
...
+    path: /etc/systemd/system/pdns.service.d/chroot-resolv.conf
+    state: absent
+  when: pdns_auth_config.chroot is not defined
+
 - name: Enable PowerDNS
   service:
     name: pdns.service
     enabled: yes
 
 - name: Start PowerDNS
```
Comment (SlothOfAnarchy) [inline tasks/main.yml:48]: You could merge these two tasks.

File: templates/pdns-chroot-resolv.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[Service]
+PermissionsStartOnly=true
+ExecStartPre=/bin/install -dm755 "{{ pdns_auth_config.chroot }}/etc"
+ExecStartPre=/bin/cp -afL /etc/resolv.conf "{{ pdns_auth_config.chroot }}/etc/resolv.conf"
```
Comment (SlothOfAnarchy) [inline templates/pdns-chroot-resolv.conf.j2:4]: Do you mean `ExecStartPost` or is it twice the same option?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: templates/pdns-chroot-resolv.conf.j2 (hunk 2 above)
Comment (dasJ) [reply templates/pdns-chroot-resolv.conf.j2:4]: It is the same option twice

File: tasks/main.yml (hunk 1 above)
Comment (dasJ) [reply tasks/main.yml:48]: COME ON. You merged this last time

Comment (SlothOfAnarchy) [reply tasks/main.yml:48]: I meant it as a suggestion. I only requested changes because of the second comment.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**